flexmeasures/data/models/time_series.py

```diff
@@ -333,22 +333,15 @@ def search_beliefs(
         :param horizons_at_least: only return beliefs with a belief horizon equal or greater than this timedelta (for example, use timedelta(0) to get ante knowledge time beliefs)
         :param horizons_at_most: only return beliefs with a belief horizon equal or less than this timedelta (for example, use timedelta(0) to get post knowledge time beliefs)
         :param source: search only beliefs by this source (pass the DataSource, or its name or id) or list of sources
-        :param most_recent_beliefs_only: only return the most recent beliefs for each event from each source (minimum belief horizon)
+        :param most_recent_beliefs_only: only return the most recent beliefs for each event from each source (minimum belief horizon). Defaults to True.
         :param most_recent_events_only: only return (post knowledge time) beliefs for the most recent event (maximum event start)
+        :param most_recent_only: only return a single belief, the most recent from the most recent event. Fastest method if you only need one.
```

Comment:
This should say something about the source (and maybe cumulative_probability). Does it return the most recent per source?

Reply:
@nhoening I think you now mentioned the cp but not yet the source.

Reply:
I improved the source parameter description because its effect is the same for all the most recent params, I believe.